addons/ofxOpenCv/scripts/formulas/opencv.sh

```diff
@@ -494,10 +582,10 @@ function build() {
 function copy() {
 
   # prepare headers directory if needed
-  mkdir -p $1/include
+  mkdir -p $1/../../addons/ofxOpenCv/libs/opencv/include
```

Comment:
Check out here: https://github.com/openframeworks/openFrameworks/blob/master/scripts/apothecary/apothecary#L512 
Add some output's and figure out if it is not settings LIBS_DIR to the addons path.